src/styles/index.scss

```diff
@@ -1,3 +1,14 @@
+@import './utils/variables';
+@import './blocks/stars';
+@import './blocks/card';
+
 body {
-  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  line-height: 18px;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0; // reset browser's default margins
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), there should not be a trailing space at the end of line 13.